functions/firebase_analytics/index.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START functions_firebase_analytics]
+
+use Google\CloudFunctions\CloudEvent;
+
+function firebaseAnalytics(CloudEvent $cloudevent)
```

Comment:
Add `void` return type for clarity

```suggestion
function firebaseAnalytics(CloudEvent $cloudevent): void
```

Reply:
Done.